.github/workflows/validate-pull-request-title/validate-pull-request-title.test.js

```diff
@@ -0,0 +1,98 @@
+import { describe, it } from "jsr:@std/testing/bdd";
+import { expect } from "jsr:@std/expect";
+import { validateTitle } from "./validate-pull-request-title.js";
+
+describe("PR Title Validation", () => {
+  const validCases = [
+    {
+      title: "feat(api): SC-1234 Add new endpoint",
+      desc: "accepts standard feature with scope and ticket",
+    },
+    {
+      title: "fix: AI-5678 Fix crash on startup",
+      desc: "accepts fix with ticket and no scope",
+    },
+    {
+      title: "chore: PT-999 Update dependencies",
+      desc: "accepts chore with PT ticket",
+    },
+    {
+      title: "revert(auth): SC-0000 Revert bad commit",
+      desc: "accepts revert with scope and ticket",
+    },
+    {
+      title: "feat(api): SC-1234-description-with-hyphens",
```

Comment:
I disagree that this should pass!

The changelog should be readable not some identifier.

This is also why I'm in favor of putting the ticket at the end, if you read the changelog the least important information is the ticket.